docs/configuration/configuration.md

```diff
@@ -23,6 +23,7 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | Option | Type | Description | Default |
 | ------ | ---- | ----------- | ------- |
 | `--acr-values` | string | optional, see [docs](https://openid.net/specs/openid-connect-eap-acr-values-1_0.html#acrValues) | `""` |
+| `--api-mode` | bool | set to true if upstream is a backend api that expects and responds json content 
```

Comment:
I think this is better described from the client point of view -- if the client expects JSON payloads in the response, right?